modules/token/client/cli/tx.go

```diff
@@ -415,3 +415,133 @@ func GetCmdSwapFeeToken() *cobra.Command {
 	flags.AddTxFlagsToCmd(cmd)
 	return cmd
 }
+
+// GetCmdSwapToErc20 implements the swap-to-erc20 command
+func GetCmdSwapToErc20() *cobra.Command {
```

Comment:
The `GetCmdSwapToErc20` subcommand should be added to the `root` command of the token module